rollup.config.js

```diff
@@ -4,12 +4,13 @@ import typescript from "@rollup/plugin-typescript";
 import dts from "rollup-plugin-dts";
 import postcss from "rollup-plugin-postcss";
 import image from '@rollup/plugin-image';
+import json from '@rollup/plugin-json'
 const packageJson = require("./package.json");
 import { terser } from "rollup-plugin-terser";
 import peerDepsExternal from 'rollup-plugin-peer-deps-external';
 import analyze from 'rollup-plugin-analyzer';
 
-const limitBytes = 1e6
+const limitBytes = 2e6
```

Comment:
Bundle size increased a lot?

I think it does not matter too much - I spent some time getting it down, but realized if downstream things are using this they would be tree-shaking the library so code they are not using shouldn't be included.  My understanding of such things is still a bit shaky though

Reply:
I'm not sure exactly where it was. At one point the markdown stuff pushed it over the limit but I'm not sure exactly how much with the latest iteration. Both of the markdown libraries pull in a lot of stuff. I'm going to try starting fresh with the package lock file and I'll check the size again after that. I did reset the package lock a couple of times already and thought it should be clean but this went through a lot of iterations working out the downstream webpack kinks so worth double checking.

Reply:
I think the real endpoint to check/verify is the bundle size when you visit https://viewlibrary.careevolutionapps.dev/ and https://viewlibrary.careevolutionapps.com/ after this is merged to see if it increased a bunch.  

Also I realize the package-lock.json flog is probably fine assuming it's not due to being on an old npm version or something...wouldn't worry too much about that...I probably should just trust semver and that any package updates that were picked up aren't breaking